projects/packages/licensing/src/class-licensing.php

```diff
@@ -217,4 +231,14 @@ public function attach_stored_licenses_on_connection() {
 			$this->attach_stored_licenses();
 		}
 	}
+
+	/**
+	 * Is the current user allowed to use the Licensing Input UI?
+	 *
+	 * @since 9.6.0
+	 * @return bool
+	 */
+	public static function is_licensing_input_enabled() {
+		return apply_filters( 'jetpack_licensing_ui_enabled', false ) && current_user_can( 'jetpack_connect_user' );
```

Comment:
Could you add a docblock here, so our parser can pick up the filter and automatically create a page in the Jetpack Codex for that new filter? Here is an example:
https://github.com/Automattic/jetpack/commit/0118cad3cc73ba0eba7635d88b1f2e9618cd75f2

Reply:
Thanks for the suggestion! I've added the docblock.

Reply:
@IanRamosC Did you forget to push this commit? 🤔